### PR TITLE
Bump Mapbox SDK services

### PR DESCRIPTION
**Pull request by DzmitryFomchyn**
### Description
Bump Mapbox Java to 7.1.0

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -23,7 +23,7 @@ ext {
 
   version = [
       mapboxMapSdk              : '10.18.1',
-      mapboxSdkServices         : '6.15.0',
+      mapboxSdkServices         : '7.1.0',
```
Comment (DzmitryFomchyn) [inline gradle/dependencies.gradle:26]: mapbox-java 6.15.0 and 7.1.0 are compatible, no breaking changes between them.

Comment (VysotskiVadim) [reply gradle/dependencies.gradle:26]: why are we bumping major version if there're no breaking changes?

Comment (DzmitryFomchyn) [reply gradle/dependencies.gradle:26]: @VysotskiVadim I believe this was done by a mistake when search team released this https://github.com/mapbox/mapbox-java/releases/tag/v7.0.0.

I think it doesn't make sense to stick v6 and duplicate minor releases for both v6 and v7 given that they're binary compatible. This shouldn't cause any issues.

Comment (VysotskiVadim) [reply gradle/dependencies.gradle:26]: wow 😨

**Review by VysotskiVadim — APPROVED** [review]

**Pull request merged**